server/app/assets/javascripts/sort_selector.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Responsible for controlling the sorting mechanism used by QuestionBankController. Elements (tagged with elementSelector class)
+ * have data attributes with the value to be compared and will sort list using the value. The data attribute is defined
+ * in QuestionSortOption.
+ *
+ * E.g. Sort by Name A-Z will look up the value for data attribute tiname-asc and sort the list by that value.
+ *
+ * @param listSelector selector for finding the portion of the document that contains the sublistSelector and elementSelector
```

Comment:
For this one it uses getElementById which takes an ID rather than a selector, so it should be named and described differently?